Cattac/Cattac/BasicLevel.swift

```diff
@@ -47,11 +39,4 @@ class BasicLevel {
         assert(row >= 0 && row < numRows)
         return grid[row, column]
     }
-    
-    func addCat(cat: Cat, _ posX: Int, _ posY: Int) {
-        let node  = nodeAtColumn(posX, row: posY)
-        node!.occupants.append(cat)
-        
-        _cats.updateValue(cat, forKey: "catID")
-    }
 }
```

Comment:
Shouldn't the func be just nodeAt? nodeAtColumn and receiving both column and row as arguments is a tad confusing